index.html

```diff
@@ -191,14 +187,55 @@ <h2>Uber Navigation</h2>
+            <span class="closeit"><i class="fa-solid fa-xmark"></i></span>
+            <div class="col-12">
+                <div class="work-main-heading px-2 py-2">
+                    <h1 class="topic-here">Tonic</h1>
+                    <ul class="list-inline flex">
+                        <li class="list-inline-item">CANOPY</li>
+                        <li class="list-inline-item li-first"><i class="fa-solid fa-circle"></i> Back End Dev</li>
+                        <li class="list-inline-item li-first"><i class="fa-solid fa-circle"></i> 2015</li>
+                    </ul>
+                    <img src="image-resources/SPortfolio.jpg" class="img-fluid w-100 " alt="Logo">
+                </div>
+            </div>
+            <div class="col-12">
+                <div class="row d-flex flex-row">
+                    <div class="col-md-8">
+                        <p class="paragh pb-2">
+                            Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releaLorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s.
+                        </p>
+                    </div>
+                    <div class=" col-md-4 d-flex flex-column">
+                        <ul class="list-inline ul-work">
+                            <li class="list-inline-item badge bg-info"><a class= "text-decoration-none text-dark" href="">Html</a></li>
+                            <li class="list-inline- badge bg-info"><a class= "text-decoration-none text-dark" href="">Css</a></li>
+                            <li class="list-inline-item badge bg-info "><a class= "text-decoration-none text-dark" href="">Javascript</a></li>
+                            <ul class="list-inline text-dark">
+                                <li class="list-inline-item badge bg-info"><a class= "text-decoration-none text-dark" href="">Bootstrap</a></li>
+                                <li class="list-inline-item badge bg-info"><a class= "text-decoration-none text-dark" href="">ruby</a></li>
+                                <li class="list-inline-item badge bg-info"><a class= "text-decoration-none text-dark" href="">github</a></li>
+                            </ul>
+                        </ul>
+                        <div class="d-flex flex-row col-md-10 gap-2 pb-2">
+                            <a href="https://abbassarwar.github.io/Abbas-Portfolio/" class="btn btn-outline-primary text-decoration-none">See Live <i class="fa-solid fa-arrow-up-right-from-square"></i></a>
+                            <a href="https://github.com/AbbasSarwar/Abbas-Portfolio" class="btn btn-outline-primary text-decoration-none">See Source <i class="fa-brands fa-github"></i></a>
```

Comment:
- [ ] The skills, See Live qnd See Source buttons are close to the left margin as per the [Figma design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=34-1143&t=Y250RJAuieEB0hor-0). This is not the case with your design. Check the screenshots below and fix this alignment issue.
- Figma design
![image](https://user-images.githubusercontent.com/62849872/223668931-be95ec47-fd10-46ac-8c61-3991550065ba.png)
- Your design
![image](https://user-images.githubusercontent.com/62849872/223669279-a9193a7e-cc86-46f0-8fb6-a472243d3e2a.png)